mobile_back_tflite/cpp/backend_tflite/tflite_c.cc

```diff
@@ -574,3 +574,7 @@ mlperf_status_t mlperf_backend_get_output(mlperf_backend_ptr_t backend_ptr,
   }
   return MLPERF_SUCCESS;
 }
+
+void *mlperf_backend_get_buffer(size_t n) { return malloc(n); }
```

Comment:
@freedomtan is this change ok?

Reply:
looks fine to me. if we have mlperf_backend_release_buffer in the right place.

Reply:
It comes to me that we didn't really have any document or header for `mlperf_backend_get_buffer() and mlperf_backend_release_buffer()`. We probably should have these documented in `flutter/cpp/c/backend_c.h`.

Reply:
These APIs are present for a long time in the app.
Because of the change in "flutter/cpp/c/dll_export.def", in this PR, the Windows CI job started throwing error. 

We can have it documented in the master branch I think.